client/blocks/comment-button/index.jsx

```diff
@@ -18,29 +18,46 @@ import { getPostTotalCommentsCount } from 'state/comments/selectors';
 
 class CommentButton extends Component {
 	static propTypes = {
-		onClick: PropTypes.func,
-		tagName: PropTypes.string,
 		commentCount: PropTypes.number,
+		link: PropTypes.string,
+		onClick: PropTypes.func,
 		showLabel: PropTypes.bool,
+		tagName: PropTypes.string,
+		target: PropTypes.string,
 	};
 
 	static defaultProps = {
-		onClick: noop,
-		tagName: 'li',
-		size: 24,
 		commentCount: 0,
+		link: null,
+		onClick: noop,
 		showLabel: true,
+		size: 24,
+		tagName: 'li',
+		target: null,
 	};
 
 	render() {
-		const { translate, commentCount, onClick, showLabel, tagName: containerTag } = this.props;
+		const {
+			commentCount,
+			link,
+			onClick,
+			showLabel,
+			tagName: containerTag,
+			target,
+			translate,
+		} = this.props;
 
 		return React.createElement(
 			containerTag,
```

Comment:
Out of curiosity how many tag types do we expect?

Reply:
Hm, great question! Nothing stops a user from entering anything they want; anything other than an `a` tag means the link won't work, which seems desirable. (Same with invalid values; the node gets created, but only the `onClick` works.)

Reply:
I've seen `li`, `div`, and `a`.